content/zh-cn/docs/reference/command-line-tools-reference/feature-gates/cpu-manager-policy-beta-options.md

```diff
@@ -0,0 +1,16 @@
+---
+title: CPUManagerPolicyBetaOptions
+content_type: feature_gate
+_build:
+  list: never
+  render: false
+---
+<!--
+This allows fine-tuning of CPUManager policies,
+experimental, Beta-quality options
+This feature gate guards *a group* of CPUManager options whose quality level is beta.
+This feature gate will never graduate to stable.
+-->
+允许对 CPUManager 策略进行微调，针对试验性的、Beta 质量级别的选项。
+此特性门控用来保护一组质量级别为 Beta 的 CPUManager 选项。
```

Comment:
I'm not sure what the current page says, but IMO it's OK to localize “CPU manager”.